doc/tutorials/sandbox_config_ui.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        class WKey(Enum):
</pre><p>I would rename the enum to SandboxConfigKeys and may exclude the save_btn.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p><code>CfgKey</code> maybe? I would have to wrap every single line if the name is that long. </p>